package.json

```diff
@@ -60,6 +61,8 @@
     "build": "cd packages/gestalt && yarn build",
     "lint:js": "eslint .",
     "lint:css": "stylelint \"packages/**/*.css\"",
+    "greenkeeper:update": "greenkeeper-lockfile-update",
```

Comment:
This seems like unnecessary indirection - why not just call `yarn greenkeeper-lockfile-update`?

Reply:
I was trying to mimic it so it looks like `lint:js` and `lint:css` but i can update